### ISSUE TITLE
Incorrect copyright headers

### ISSUE THREAD

**Issue by Adamantios** [valory-xyz/open-autonomy#2204]
The copyright is incorrect in multiple files.

This was not detected because the CI does not check the copyright correctly [here](https://github.com/valory-xyz/open-autonomy/blob/v0.14.10/.github/workflows/main_workflow.yml#L92).

**Comment by Devanshusisodiya** [comment]
@Adamantios The copyright info for example in the [file](https://github.com/Devanshusisodiya/open-autonomy/blob/3b0b1a2135215e7e6e70fab73a0e6fcc282343b5/scripts/__init__.py#L4) should be updated to `Copyright 2024 Valory AG` and similarly everywhere?

**Comment by Adamantios** [comment]
@Devanshusisodiya no, the copyright of a file should have a range specifying when it was created and when it was last updated, or a single number if the year of creation and last update are the same.

The linked command on the CI above should be replaced with:
```
run: tomte check-copyright --author valory --author fetchai
```

Running this locally should also give hints on what the copyright issues currently are.

**Comment by Devanshusisodiya** [comment]
@Adamantios I have raised this [PR](https://github.com/valory-xyz/open-autonomy/pull/2208) to fix this issue, please review and comment. Thanks.
